kolibri/core/assets/src/views/breadcrumbs/index.vue

```diff
@@ -1,43 +1,64 @@
 <template>
 
-  <nav class="breadcrumbs">
-
-    <div v-if="collapsedCrumbs.length" class="breadcrumbs-dropdown-wrapper">
-      <ui-icon-button :has-dropdown="true" icon="expand_more" size="small">
-        <div slot="dropdown" class="breadcrumbs-dropdown">
-          <ol class="breadcrumbs-dropdown-items">
-            <li v-for="crumb in collapsedCrumbs" class="breadcrumbs-dropdown-item">
-              <router-link :to="crumb.link" :style="{ maxWidth: `${collapsedCrumbMaxWidth}px` }">
-                {{ crumb.text }}
-              </router-link>
-            </li>
-          </ol>
-        </div>
-      </ui-icon-button>
+  <div>
+    <nav class="breadcrumbs">
+      <div v-if="collapsedCrumbs.length" class="breadcrumbs-dropdown-wrapper">
```

Comment:
this might be better as a `v-show` since the condition is liable to change at runtime